portal-impl/src/portal.properties

```diff
@@ -7636,11 +7636,18 @@
 ##
 
     #
-    # Set this to true to enable minification of CSS and JavaScript resources.
+    # Set this to true to enable run-time minification of CSS and JavaScript
+    # resources by the com.liferay.portal.minifier.MinifierUtil class. Note that
+    # even when run-time minification is off, build-time minification of
+    # frontend assets (JS and CSS) may still occur, as dictated by the
+    # "nodejs.node.env" build property. Specifically, when set to
+    # "development", build-time minfication does not take place; when set to
+    # any other value, including the default value of "production", build-time
+    # minification is performed.
```

Comment:
Based on questions in Slack, it seems I could make this clearer.

```
# ...
# as dictated by the "nodejs.node.env" setting in build.properties.
# Specifically, when "nodejs.node.env" is set to "development",
# ...
```

Reply:
I did get this, but I see how it can be confusing

Reply:
Also, I'm not sure if it should go here, because customers will see the `portal.properties` commentary AFAIK, but they don't know anything about our build... :thinking:

Reply:
However, I'm not sure how this file is related to the final delivered product... I mean, I don't know exactly how customers know about the `portal.properties` entries and their use. But in any case I think `portal.properties` is a pure runtime artifact IIANM

Reply:
> However, I'm not sure how this file is related to the final delivered product... I mean, I don't know exactly how customers know about the portal.properties entries and their use. But in any case I think portal.properties is a pure runtime artifact IIANM

They copy properties from here into their own `portal-ext.properties` or other mechanisms.

Reply:
In that case I would cut the comment to just this parte:

# Set this to true to enable run-time minification of CSS and JavaScript
# resources by the com.liferay.portal.minifier.MinifierUtil class. Note that
# even when run-time minification is off, build-time minification of
# frontend assets (JS and CSS) may still occur.

And add something like:

# In fact, that's the preferred way to do it since 7.4.

Or something like that...

Also, we are adding notices like this:

>     # This property is deprecated and will be removed in a future release.

To deprecated properties in this file (f.e: [here](https://github.com/liferay/liferay-portal/blob/5aceff3ec63a5118178d7967f0903964b769a773/portal-impl/src/portal.properties#L3261) and [here](https://github.com/liferay/liferay-portal/blob/5aceff3ec63a5118178d7967f0903964b769a773/portal-impl/src/portal.properties#L1714).

Reply:
> And add something like:
> 
> ```
> # In fact, that's the preferred way to do it since 7.4.
> ```
> 
> (or something like that...)

I like this suggestion, and will claim it as my own, making it something like:

```
# In fact, that's the preferred way to do it since 7.4 (hash tag: "JUST SAYIN").
```

Reply:
Updated with suggested edits in https://github.com/liferay-frontend/liferay-portal/pull/553/commits/20c8f8da77333f63e4145ecb7b92d58bcee5d252